flow/include/flow/error_definitions.h

```diff
@@ -137,6 +137,7 @@ ERROR( proxy_tag_throttled, 1223, "Exceeded maximum proxy tag throttling duratio
 ERROR( key_value_store_deadline_exceeded, 1224, "Exceeded maximum time allowed to read or write.")
 ERROR( storage_quota_exceeded, 1225, "Exceeded the maximum storage quota allocated to the tenant.")
 ERROR( audit_storage_error, 1226, "Found data corruption" )
+ERROR( master_failed, 1227, "Cluster recovery terminating because master has failed")
```

Comment:
It is true that `master_failed` triggers the cluster recovery, but not sure if this is the case here. 
Though I see that you kept the description consistent with `tlog_failed` , so may be its fine.